backend/src/routes/api/rolebindings/index.ts

```diff
@@ -41,8 +41,13 @@ module.exports = async (fastify: KubeFastifyInstance) => {
         );
         return rbResponse.body;
       } catch (e) {
-        fastify.log.error(`rolebinding could not be created: ${e}`);
-        reply.send(e);
+        if (e.response?.statusCode === 409) {
+          fastify.log.warn(`Rolebinding already present, skipping creation.`);
+          return {};
+        } else {
+          fastify.log.error(`rolebinding could not be created: ${e}`);
+          reply.send(e);
```

Comment:
Can we look to log `e.response.body.message` instead of just `e`... also could we create a new Error object and send that instead? We do too many logging of just `e` or `e.toString()` and it serializes poorly into the logs. 

When a request fails through the kube library, there is a status object at `e.response.body` -- this is the K8s response to the request. `.message` on that gives back what k8s saw happen.

Reply:
Ok changed it to new error object and message